pop-api/src/lib.rs

```diff
@@ -34,7 +34,12 @@ mod constants {
 }
 
 /// Helper method to build `ChainExtensionMethod``
-pub fn build_extension_method(
+///
+/// - `version`: The version of the chain extension
+/// - `function`: The ID of the function
+/// - `module`: The index of the runtime module
+/// - `dispatchable`: The index of the module dispatchable functions
+fn build_extension_method(
```

Comment:
Please document like:

```
/// <Description>.
///
/// Parameters:
/// - '<parameter>': <description>.
```

Reply:
This is inconsistent in the repo and will be fixed (#105), my apologies.

Reply:
@Daanvdplas So should we ignore it and make a refractor in the other PR?

Reply:
No you can update here already